src/main/resources/static/js/main.js

```diff
@@ -87,6 +87,18 @@ $(document).ready(function() {
     //-----------------experiment page---------------------
     // tooltip hover
     $('[data-toggle="tooltip"]').tooltip();
+
+	// prevent start and delete button from multi clicks
+	$(".click-once-button").click(function() {
+		$(".click-once-button").prop("disabled", true).addClass("disabled");
+		// TODO: refresh the page after click
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS1135)